code/game/dna/mutations/disabilities.dm

```diff
@@ -241,7 +241,7 @@
 	desc = "Completely shuts down the speech center of the subject's brain."
 	activation_messages = list("You feel unable to express yourself at all.")
 	deactivation_messages = list("You feel able to speak freely again.")
-	instability = -GENE_INSTABILITY_MODERATE
+	instability = -GENE_INSTABILITY_MAJOR
```

Comment:
Mute missing from the PR description i think

Reply:
Good catch, I'll add it in must've missed it in the big list.